tox.ini

```diff
@@ -4,17 +4,15 @@
 # When updating the envlist, be sure to also update TOX_ENV in .travis.yml
 envlist =
     docs,
-    {py27,py34}-django{1.6,1.7,1.8}-drf{3.0,3.1,3.2}
+    {py27,py34,py35}-django{1.8,1.9}-drf{3.2,3.3}
```

Comment:
To add a single combo as I mentioned earlier, you would add a new line to this section, like:

    py27-django1.6-drf3.3